arm-software/docs/arm-toolchain-features.md

```diff
@@ -19,4 +19,11 @@ There are no experimental features implemented.
 
 # Features
 
-There are no additional features implemented.
+## Additional loop unroll in the LTO pipeline
+In some cases it is benefitial to perform an additional loop unroll pass so that extra information becomes available to later passes, e.g. SROA.
```

Comment:
```suggestion
In some cases it is beneficial to perform an additional loop unroll pass so that extra information becomes available to later passes, e.g. SROA.
```